src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -91,7 +92,8 @@ public class ReportPortalExtension
 	private final Map<ExtensionContext, Maybe<String>> idMapping = new ConcurrentHashMap<>();
 	private final Map<ExtensionContext, Maybe<String>> testTemplates = new ConcurrentHashMap<>();
 	private final Set<ExtensionContext> failedClassInits = Collections.newSetFromMap(new ConcurrentHashMap<>());
-
+	public static final String DESCRIPTION_TEST_ERROR_FORMAT = "%s\nError: \n%s";
+	private Throwable testThrowable;
```

Comment:
No, that's not going to work for parallel execution, each thread will rewrite this field, please take it into account.